.claude/scripts/work-priority-ladder.test.sh

```
@@ -138,6 +154,278 @@ assert_prose 'A cap is NOT licence to stop early, and it never blocks the floor'
+assert_prose 'That exception is time-bounded, never a standing state' \
+  "${constitution_flat}" "floor exception does not state that it expires — it reads as permanent"
+assert_prose 'no undefined permanent-sounding gate' \
+  "${constitution_flat}" "floor exception does not forbid an undefined gate standing in for a signal"
+
+# ── one author matrix for `--auto`, with the conditional fourth author named ───────────────────────
+# `--auto` merges whatever head passes checks LATER, so eligibility is a question about the AUTHOR.
+# Naming only three Apps while a separate clause granted exit-0 programmed updater PRs the no-review
+# auto path left `app/botantler-1` unplaced: the maintenance overlay's "non-qualifying" wording
+# implies a QUALIFYING one is eligible, but no surface said so outright. Pin the four-author form and
+# the condition, so a later edit cannot silently drop back to the ambiguous bare list.
+assert_absent '(`github-actions`/`ksail-bot`/`app/cursor`) uses pre-CLEAN' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)